### PR TITLE
Change binormal to bitangent

### LINKED ISSUE #207: Finish binormal to bitangent rename
Once https://github.com/AnalyticalGraphicsInc/cesium/issues/4822 is completed.

See
* https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/generateTangentsBitangents.js#L44
* https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/cesiumGeometryToGltfPrimitive.js#L68
* https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/gltfPrimitiveToCesiumGeometry.js#L52

### PR DESCRIPTION
**Pull request by shehzan10**
Fixes #207 

Requires https://github.com/AnalyticalGraphicsInc/cesium/commit/e440efcc4e93427e8f31db131af09bd8d4de0469. So this might fail until the `package.json` file is updated.

Ps. includes trailing whitespace fixes for modified files.

### CODE REVIEW

**Comment by pjcozzi** [comment]
@mlimper would you mind doing a quick review here?

We'll fully review this PR then park it until the Cesium 1.30 release is posted to npm.

File: lib/gltfPrimitiveToCesiumGeometry.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ function mapPrimitiveAttributeToGeometry(gltf, primitive, semantic, geometryAttr
     } else if (semantic.indexOf('BITANGENT') === 0) {
         //TODO: refactor this case soon as it got renamed in Cesium
```
Comment (mlimper) [inline lib/gltfPrimitiveToCesiumGeometry.js]: this TODO can also be removed

**Comment by mlimper** [comment]
Looks good - thanks @shehzan10, also for fixing the formatting errors!

I'd, nevertheless, suggest to try it out using a locally updated Cesium package before changing the `package.json` and doing the merge - but I guess that's just stating the obvious ;)

**Comment by pjcozzi** [comment]
Thanks for the prompt review, @mlimper!

@shehzan10 let us know when this is ready.

File: lib/gltfPrimitiveToCesiumGeometry.js (hunk 1 above)
Comment (shehzan10) [inline lib/gltfPrimitiveToCesiumGeometry.js]: Resolved

**Comment by shehzan10** [comment]
@pjcozzi this PR is ready now.

@mlimper Yes, I did test it by using a local package. Do you want me to push a commit with a temporary `package.json` update so that CI can pass, and then remove it before the final merge?

**Comment by mlimper** [comment]
> @mlimper Yes, I did test it by using a local package. Do you want me to push a commit with a temporary package.json update so that CI can pass, and then remove it before the final merge?

You could do so, or maybe just wait until Cesium is ready and then check if CI passes before merging this PR.

**Comment by shehzan10** [comment]
> You could do so, or maybe just wait until Cesium is ready and then check if CI passes before merging this PR.

Yes, the idea was to wait.

**Comment by pjcozzi** [comment]
Also LGTM.  @shehzan10 please bump this PR when the Cesium 1.30 release is out on Feb 1.

**Comment by shehzan10** [comment]
@pjcozzi Updated the `package.json` to use Cesium v1.30. The CI passes now.

**Pull request merged**